document/core/exec/numerics.rst

```diff
@@ -173,6 +327,13 @@ The integer result of predicates -- i.e., :ref:`tests <syntax-testop>` and :ref:
    \bool(C) &=& 0 & (\otherwise) \\
    \end{array}
 
+We lift :math:`\bool` to sequences of predicates by conjunction over the inputs.
+
+.. math::
+   \begin{array}{lll@{\qquad}l}
+   \bool(C_1~C_2^*) &=& \bool(C_1) \wedge \bool(C_2^*)
```

Comment:
Do we need to define how `\bigwedge` works?

Reply:
We could call it out as more auxiliary notation in syntax/conventions, like so:
```
* :math:`\bigwedge s^\ast` and :math:`\bigvee s^\ast` denote the conjunction and disjunction of a sequence of conditions, respectively; if the sequence is empty, then the result is vacuously true, respectively false.
```
OTOH, it's more or less established math notation and fairly intuitive, so I suppose it would be understandable without extra explanation. WDYT?

Reply:
I agree that it's established notation and fairly intuitive, especially for someone reading formal notation. I'll use `\bigwedge` directly, thanks!